stable/kube-state-metrics/Chart.yaml

```
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 1.3.0
+version: 1.5.1
```

This has to be changed again. Sorry :(.

```suggestion
version: 1.6.1
```